manual/python/index.md

```diff
@@ -144,29 +151,29 @@ Moreover, you can use `addressof` in conjunction with `TTree::Branch` from Pytho
 
 - `BindObject`: Equivalent to [cppyy's `bind_object`](https://cppyy.readthedocs.io/en/latest/lowlevel.html#c-c-casts){:target="_blank"}.
 
-- `MakeNullPointer(class_proxy)`: Equivalent to `BindObject(0, class_proxy)`, it returns a Proxy object of the class represented by `class_proxy` that is bound to a C++ null pointer. For example, `MakeNullPointer(TTree)` returns a `TTree` proxy object which internally points to null. This function is kept for backwards compatibility with old PyROOT versions.
+- `MakeNullPointer(class_proxy)`: Equivalent to `BindObject(0, class_proxy)`, it returns a proxy object of the class represented by `class_proxy` that is bound to a C++ null pointer.<br/>For example, `MakeNullPointer(TTree)` returns a `TTree` proxy object that internally points to null. This function is kept for backwards compatibility with old PyROOT versions.
 
-- `SetOwnership(obj, python_owns)`: A Python proxy can either own or not own its internal C++ object. If a Python proxy owns its C++ object and the proxy is being destroyed, the C++ object will be deleted too. This ownership can be modified for a given Python proxy with `SetOwnership`. For example, when calling `SetOwnership(obj, False)`, the user makes sure that the C++ object proxied by `obj` will not be deleted by PyROOT when `obj` is garbage collected; this is useful for example, if the user knows the deletion will happen in C++ and wants to prevent a double delete. This functionality should be used with care not to produce memory leaks.
+- `SetOwnership(obj, python_owns)`: A Python proxy can either own or not own its internal C++ object. If a Python proxy owns its C++ object and the proxy is being destroyed, the C++ object is deleted too. This ownership can be modified for a given Python proxy with `SetOwnership`.<br>For example, when calling `SetOwnership(obj, False)`, make sure that the C++ object proxied by `obj` is not deleted by PyROOT when `obj` is garbage collected. This is useful for example, if you know that the deletion will happen in C++ and you want to prevent a double delete.<br/>Use this functionality with care not to produce any memory leaks.
 
 ## Loading user libraries and Just-In-Time compilation (JITting)
 
 With PyROOT you can use *any C++ library* from Python, not only the ROOT libraries. This is possible because of the automatic and dynamic bindings between Python and C++ that PyROOT provides.
 Without any prior generation of wrappers, at execution time, PyROOT can load C++ code and call into it.
 
-This allows for writing high-performance C++, compiling it, and using it from Python.
+This enables you to write high performance C++, compile it and use it from Python. 
 The following options are available, ordered by complexity and performance:
```

Comment:
I think that the fact that the options are ordered by complexity and performance justifies the numbering below. (instead of just bullets).

Reply:
An ordered list is only used, if there is a sequence (especially in tasks). Complexity (which means for everybody something else) is no real ordering criteria (in the Technical Documentation).

if you only scan the menu on the left side, it is assumed that there is a sequence to be followed:
   Loading user libraries and Just-In-Time compilation (JITting)
     1. Just-in-time compilation ...
     2. Just-in-time compilation ..
     3. Loading ...
     4. Loading ...

So in this case a numbered list is not correct.